include/GafferUI/StandardConnectionGadget.h

```diff
@@ -92,6 +92,11 @@ class StandardConnectionGadget : public ConnectionGadget
 
 		bool updateUserColor();
 
+		bool __dropDot( Imath::V2f position );
+		void __updateDotPreviewBounds( const ButtonEvent &event );
+		bool __keyPressed( const KeyEvent &event );
```

Comment:
I wasn't sure whether to consider these private to the DotAdderThingy implementation or proper class methods without the double underscores.